docs/examples/forward_ref.py

```diff
@@ -4,12 +4,14 @@
 Foo = ForwardRef('Foo')
 
 class Foo(BaseModel):
-    a: int = 123
-    b: Foo = None
+    a: int = None
+    b: Dict[str, FooType] = {}
```

Comment:
No need to change the example, unless otherwise required examples should be as simple as possible.